homeassistant/components/climate/ecobee.py

```diff
@@ -269,6 +271,7 @@ def device_state_attributes(self):
             "fan": self.fan,
             "mode": self.mode,
             "operation": operation,
+            ATTR_AUX_HEAT: aux_heat,
```

Comment:
@balloob: The implementor wanted to have an attribute that tracked when ecobee was in auxilliary heating mode. As you point out, there is indeed an attribute in the climate device `ATTR_AUX_HEAT` which appears to reflect this mode. In ecobee this mode is enabled differently from the climate device. There are no turn on or turn off services for auxiliary mode; instead, this mode is enabled as part of the normal operation mode of the device. I suspect this would be the case in most climate devices and that the `SERICE_SET_AUX_HEAT` in the climate device is not really needed. In any case, the implementor did not want to interfere with this possibly different use of the auxiliary heat attribute. However, I now agree with you that ecobee should hijack that attribute, but leave the related services unimplemented.